src/tm/class-wpml-tm-page-builders-field-wrapper.php

```diff
@@ -93,14 +93,25 @@ public function get_string_type() {
 		return $result;
 	}
 
+	/**
+	 * Get string wrap.
+	 *
+	 * @param stdClass $string WPML string.
+	 *
+	 * @return string
+	 */
+	public static function get_wrap( $string ) {
```

Comment:
This methods looks overkill @kagg-design.

Reply:
@strategio maybe, but I wanted to make code in WPML_Page_Builders_Register_Strings::register_strings_for_node() uniform. Also, method can be extended in the future.